### PR TITLE
Remove Markdown link, add period to plugin description

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "com.mattermost.aws-sns",
     "name": "AWS SNS",
-    "description": "Send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS",
+    "description": "Send alert notifications from Amazon AWS CloudWatch (https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS.",
```
Comment (lieut-data) [inline plugin.json]: nit: do we need to link out to CloudWatch at all? Seems like this is something a customer can Google for themselves (along with "AWS SNS" if that's foreign to them).

```suggestion
    "description": "Send alert notifications from Amazon AWS CloudWatch to Mattermost channels via AWS SNS.",
```

File: plugin.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         }
     },
     "settings_schema": {
-        "header": "",
+        "header": "This plugin is used to send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS",
```
Comment (lieut-data) [inline plugin.json]: 🎉 -- this seems like a much better solution overall. It also looks like could stand to document which of these fields support Markdown today -- I'll review the code and submit an update to the documentation.

Comment (lieut-data) [reply plugin.json]: Minor nit: should this end with a period?
```suggestion
        "header": "This plugin is used to send alert notifications from [Amazon AWS CloudWatch](https://aws.amazon.com/cloudwatch/) to Mattermost channels via AWS SNS.",
```

Comment (cpanato) [reply plugin.json]: Just a small comment. The plugin does not send any alerts it receives alert from sns. Maybe can update this part

**Pull request merged**